### PR TITLE
PIL-1051 Storyly improvements: Change Storyly container shape to rounded square

### PR DESCRIPTION
**Pull request by divyaBacancy**
StorySize-IOS
StorySize-android

### CODE REVIEW

**Comment by IAmKio** [comment]
@divyaBacancy are we missing the defaults?

File: src/constants/remoteConfigConstants.js [hunk 1]
```diff
@@ -40,6 +40,12 @@ export const REMOTE_CONFIG = {
   FEATURE_ONBOARDING: 'feature_onboarding',
   ENS_MIGRATOR_ENABLED: 'ens_migrator_enabled',
   KNOWLEDGE_BASE_URL: 'knowlegebase_url',
+  ANDROID_FEATURE_STORIES_HEIGHT: 'feature_stories_android_height',
```
Comment (IAmKio) [inline src/constants/remoteConfigConstants.js:43]: Could we keep the naming consistent please with the feature flag name?
e.g
```suggestion
  FEATURE_STORIES_ANDROID_HEIGHT: 'feature_stories_android_height',
```

File: src/constants/remoteConfigConstants.js [hunk 2]
```diff
@@ -62,4 +68,10 @@ export const INITIAL_REMOTE_CONFIG = {
   [REMOTE_CONFIG.FEATURE_ONBOARDING]: false,
   [REMOTE_CONFIG.ENS_MIGRATOR_ENABLED]: false,
   [REMOTE_CONFIG.KNOWLEDGE_BASE_URL]: 'https://help.pillarproject.io/',
+  ANDROID_FEATURE_STORIES_HEIGHT: '200',
```
Comment (IAmKio) [inline src/constants/remoteConfigConstants.js:71]: Thanks for adding this! 🙏

**Review by IAmKio — CHANGES_REQUESTED** [review]

File: src/constants/remoteConfigConstants.js (hunk 1 above)
Comment (divyaBacancy) [reply src/constants/remoteConfigConstants.js]: Done.

**Pull request merged**